sunpy/map/tests/test_mapbase.py

```diff
@@ -284,6 +289,25 @@ def test_rotation_matrix_cd_cdelt_square():
     np.testing.assert_allclose(cd_map.rotation_matrix, np.array([[0., -1], [1., 0]]))
 
 
+def test_norm_cmap_kwargs(generic_map):
```

Comment:
Could possibly have a better name, but `kwargs` didn't seem appropriate given the above changes.

```suggestion
def test_norm_cmap_setters(generic_map):
```